src/main/java/com/parth/importer/services/StudentService.java

```diff
@@ -0,0 +1,61 @@
+package com.parth.importer.services;
+
+import com.parth.importer.dto.LogDisplayDto;
+import com.parth.importer.dto.StudentAdditionDto;
+import com.parth.importer.dto.StudentDisplayDto;
+import com.parth.importer.mapstructMapper.LogMapper;
+import com.parth.importer.model.Log;
+import com.parth.importer.repository.LogRepository;
+import org.apache.ibatis.javassist.NotFoundException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.*;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import org.springframework.web.client.HttpClientErrorException;
+import org.springframework.web.client.HttpServerErrorException;
+import org.springframework.web.client.RestTemplate;
+
+import java.nio.file.AccessDeniedException;
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+public class StudentService {
+
+    @Autowired
+    LogRepository logRepository;
+
+    @Autowired
+    LogMapper logMapper;
+
+    private static String POST_STUDENTS_URL = "http://localhost:8081/students";
+
+    public List<LogDisplayDto> addStudents(List<StudentAdditionDto> studentAdditionDtos, String token) {
```

Comment:
```suggestion
    public List<LogDisplayDto> addStudents(List<StudentAdditionDto> studentAdditionDtos) {
```
Retrieve token from security context holder

Reply:
Done